source/includes/stackpath/_certificates.md

```diff
@@ -0,0 +1,52 @@
+## Certificates
+
+<!-------------------- CREATE A CERTIFICATE -------------------->
+
+### Create a certificate
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/certificates?siteId=f9dea588-d7ab-4f42-b6e6-4b85f273f3db"
+```
+> Request body example for creating a certificate:
+
+```json
+{
+  "certificate": "-----BEGIN CERTIFICATE----- X509 certificate content -----END CERTIFICATE-----",
+  "key": "-----BEGIN PRIVATE KEY----- Private key of the certificate -----END PRIVATE KEY-----",
+  "caBundle": "-----BEGIN CERTIFICATE-----Certificate authority bundle  -----END CERTIFICATE-----"
```

Comment:
Missing space between `-----BEGIN CERTIFICATE-----` and `Certificate authority bundle`. 

Also is there a reason we don't put an example for the `X509 certificate content` and `Private key of the certificate` instead of putting those placeholders?

Reply:
I felt like putting certificate and private key could be security issue as the document is accessible.

Reply:
Added the space.
https://github.com/cloudops/cloudmc-api-docs/pull/351/commits/7bde375ac420701f38fc00193d82eb01c60db549